FAQ-LICENSING.md

```diff
@@ -0,0 +1,47 @@
+# JTS License FAQ
+
+JTS is dual-licensed under the EDL and EPL 
+as described [here](LICENSES.md).
+
+This FAQ clarifies the implications of the JTS software licensing.
+
+## Terminology
+
+* **Downstream projects** are codebases which link to JTS libraries
+  * e.g. GeoTools, GeoServer, etc.
+* **Derivative projects** are ports of JTS to other languages 
+  * e.g. GEOS, JSTS, etc.
+
+
+## Licensing Frequently Asked Questions
+
+**A1. What is the difference between EDL and BSD-3?**
+
+  BSD-3 is a license *family*.  EDL is the actual license and is called that because Eclipse is the distributing organization.
+
+**A2. JTS no longer uses LGPL.  What does this mean for downstream projects?**
+
+JTS is now released under a dual license: EDL and EPL.  
+Dual-licensing gives downstream projects more options:
+
+* The **EDL** is a BSD-3-style license, which is more permissive than the LGPL.  
+  This allows a wider variety of uses; for instance, it permits iOS use of JTS for mobile development.
+  
+* The **EPL** provides a similar permissiveness to LGPL with different wording around patents
+
+**A3. But I thought LGPL and EPL do not mix? How does this effect derivative projects (like the LGPL GEOS project)?**
+
+An LGPL project (like GEOS) should use a BSD-3-style License (the BSD-3 License is compatible with the LGPL license used by GEOS).
+
+In the longer term the EPL license is being revised to work with GPL / LGPL.  
+This will allow derivative projects like GEOS the choice of using either BSD-3-style or EPL license.
```

Comment:
Would it be accurate to replace "BSD-3-style" with "EDL" ?

Reply:
Yes, seems more accurate.  Although it's still a bit confusing then, because it seems to suggest that GEOS should change to use EDL or EPL, which I don't think is the intention.  

Perhaps it should probably say "*New* derivative projects" - and not mention GEOS?

Reply:
Yes, I think it could be clarified by again replacing "the choice of using using either..." with "the choice of using JTS under the terms of either... "

Agree that the FAQ doesn't need to mention GEOS a second time here.